cpp/tests/test_shuffler.cpp

```diff
@@ -20,6 +20,7 @@
 #include <rapidsmpf/utils.hpp>
 
 #include "environment.hpp"
+#include "rapidsmpf/buffer/buffer.hpp"
```

Comment:
```suggestion
#include <rapidsmpf/buffer/buffer.hpp>
```

And then move it up.

Reply:
done